docs/DevelopmentPlan/DevelopmentPlan.tex

```diff
@@ -14,39 +15,104 @@
+	\begin{itemize}
+		\item Entire test run has to be successful to ensure \emph{develop} is in a healthy state
+		\item Test coverage delta must not be reduced unless redundant tests are taken out
+		\item Spotbugs must pass, enforcing healthy code practices
+		\item Snyk checks must pass, ensuring the packages used do not have vulnerabilities
+	\end{itemize}
+	\item Each PR must have at minimum a description filled out, and a relevant feature ticket must have a Jira issue attached along with it
+	\item If a feature has an attached ticket, the PR title must start with [CAP-\#\#] and then a title
+	\item Every time a PR is made, automation will ping \#{}pr-bot on Discord so the team is aware of changes being made
+\end{itemize}
 
+Issue Tracking - \href{https://flickpicker.atlassian.net/jira/software/projects/CAP/boards/1}{Jira}: 
 \begin{itemize}
-	\item How will you be using git, including branches, pull request, etc.?
-	\item How will you be managing issues, including template issues, issue
-	classificaiton, etc.?
+	\item All development changes need a descriptive ticket cut before it is ready for code review. Automation will link the PR to the ticket and vice versa as well
+	\begin{itemize}
+		\item Descriptive means the ticket must have a title and acceptance criteria (AC) to complete the ticket
+	\end{itemize}
+	\item The status of the ticket must be updated on the board. Most importantly, if it is in ``To Do" so multiple developers do not start working on the same feature
+	\item Points will be arbitrary for the ticket, based on how much work the developer is working on it thinks it will be. It is going to be an indication of how complex the work is for the reviewers
+	\item Utilize Jira's ticket types to have issue classifications:
+	\begin{itemize}
+		\item Story: Ticket describing a new feature
+		\item Bug: Fixing existing code
+		\item Task: Small changes that do not fall in either of the above categories
+	\end{itemize}
 \end{itemize}
 
+\subsection{Example Feature Workflow}
+An example developer workflow for a feature will be as follows:
+\begin{enumerate}
+	\item Jira ticket is cut with a description and AC, assigned to a developer, then the ticket is moved to ``In Progress" while it is developed
+	\item Branch is made for development and then marked ``Ready for Review" when the owner thinks the AC is met
+	\item Reviewers ensure AC is met and healthy coding practices are followed
+	\item PR is merged, and the ticket is moved to ``Done"
+\end{enumerate}
+
+\subsection{Example Documentation Workflow}
+An example documentation workflow will be as follows:
+\begin{enumerate}
+	\item Asynchronous discussion on Discord is done to choose which sections of a documentation to update
+	\item Branch is made for updates and then marked ``Ready for Review" when the owner thinks the documentation has been sufficiently written
+	\item PR is merged
+\end{enumerate}
+
```

Comment:
LGTM, there's a couple spots that may need rewording, but I'll add those in myself as I gotta work on the other sections anyway